### PR TITLE
Zonership readme changes

### PR DESCRIPTION
**Pull request by ldunnplaymint**
We are now using a -z flag to specify zones in the ds cli, need to update the documentation to reflect that

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,11 +136,16 @@ For `make` builds (and OSX docker builds) the MAP variable can be set as part of
 MAP=quest-map make dev
 ```
 
-## 2. Apply a map after deploying
+## 2. Claiming a zone
+
+Once your build has succeeded, `http://localhost:3000/tile-fabricator` will take you to the Downstream homepage. Here you can sign in via Metamask, Wallet connect or use one of our Burner wallets. When deploying locally, a wallet called the "LocalDevAccoint" will already own Zone 1. You can connect using said wallet to speed things up. If you want to claim a new one using any other login method, make sure to note down the Zone Number as you will need to pass it through our `ds cli` tool using the `-z` flag.
```
Comment (chrisfarms) [inline README.md:141]:
```suggestion
Once your build has succeeded, `http://localhost:3000/` will take you to the Downstream homepage. Here you can sign in via Metamask, Wallet connect or use one of our Burner wallets. When deploying locally, a wallet called the "LocalDevAccoint" will already own Zone 1. You can connect using said wallet to speed things up. If you want to claim a new one using any other login method, make sure to note down the Zone Number as you will need to pass it through our `ds cli` tool using the `-z` flag.
```

File: contracts/src/maps/tutorial-room-1/README.md [hunk 2]
```diff
@@ -153,14 +158,15 @@ We will use the Downstream CLI to deploy our newly created building to our local
...
+- Still using Zone 1 (`-z 1`)
 - `ds apply -n local -z 1 -k <private-key> -f BasicFactory.yaml`
 
 You should see the terminal output display for the building kind and item kind defined by your new factory:
 
 <img src="./readme-images/step6a.png" width=200>
 
 This type of building now exists in our local instance of Downstream, which means it is available for Units to build on the map:
-- Browse to [localhost:3000]([http://localhost:3000]).
+- Browse to [localhost:3000]([http://localhost:3000/zones/1]).
```
Comment (chrisfarms) [inline contracts/src/maps/tutorial-room-1/README.md:169]:
```suggestion
- Browse to [http://localhost:3000/zones/1].
```

**Comment by ldunnplaymint** [comment]
This is probably the bare minimum to update the docs with zones. I am sure there is more attention needed but I will get to that next.

**Review by chrisfarms — APPROVED** [review]
some of the markdown links look wrong to me ... but they are wrong in the original too so guess github is being forgiving about rendering them

**Pull request merged**